Makefile

```diff
@@ -1,33 +1,20 @@
-NAME	= wmidump
+NAME=	wmidump
 
-CC	?= gcc
```

Comment:
This just undoes an earlier commit, so just drop it there instead of changing it and then deleting it.